code/parachain/frame/pablo/src/weights.rs

```diff
@@ -21,4 +22,5 @@ impl WeightInfo for () {
 	fn remove_liquidity() -> Weight {10_000}
 	fn buy() -> Weight {10_000}
 	fn swap() -> Weight {10_000}
+	fn do_create_pool() -> Weight {10_000}
```

Comment:
weird naming prefix `do_`

Reply:
It's benching `do_create_pool` specifically